### ISSUE TITLE
how to pipe with dwn?

### ISSUE THREAD

**Issue by st3rven** [sensepost/dwn#6]
Hello, I find your project very interesting, so thanks in advance.

I have a question, how could I make use of "pipe" to pass data between tools using dwn?

considering that one of the plans is gau, regularly with gau we could do this:
`printf example.com | gau`
or even concatenate with multiple tools that they will be gradually integrated and support STDIN with pipe.. so, I think that this is currently a limitation of the project or maybe I just don't know how to make it possible with dwn.

**Comment by leonjza** [comment]
Yeah i think std{in,out} redirection won't work great yet. Will have to fiddle with the dockerpy options to get that right. I don't think it's impossible though.

**Comment by st3rven** [comment]
cool, hope you can provide a solution to this. 
Thanks @leonjza
